src/apps/backend/modules/access_token/access_token_service.py

```diff
@@ -40,8 +40,8 @@ def create_access_token_by_phone_number(*, params: OTPBasedAuthAccessTokenReques
 
     @staticmethod
     def __generate_access_token(*, account: Account) -> AccessToken:
-        jwt_signing_key = ConfigService.get_token_signing_key()
-        jwt_expiry = timedelta(days=ConfigService.get_token_expiry_days())
+        jwt_signing_key = ConfigService.get_value(key="TOKEN_SIGNING_KEY", section="ACCOUNTS")
+        jwt_expiry = timedelta(days=int(ConfigService.get_value(key="TOKEN_EXPIRY_DAYS", section="ACCOUNTS")))
```

Comment:
ConfigService should expose get_int, get_bool etc and enforce type safety instead of each consumer know how to do the conversion.